docs/DeveloperGuide.md

```diff
@@ -416,18 +416,23 @@ There are currently 4 custom predicates implemented in FAST:
 `TagMatchesKeywordPredicate` implemented by running the tags through a for-loop and checking if any of them match the given keyword.
 `RemarkContainsKeywordPredicate` implemented by using the inbuilt `String::contains`.
 
+![Find_Command_Class_Diagram](images/findcommandpredicates.png)
```

Comment:
I feel that this diagram is not accompanied by a proper explanation. The part which shows that the predicates implement the Predicate interface is good! Perhaps you can elaborate more about the `test(person, Persons)` method so that the diagram is more relevant